pyproject.toml

```diff
@@ -15,7 +15,7 @@ pyyaml = "^6.0.2"
 pykube-ng = "^23.6.0"
 munch = "^4.0.0"
 prometheus-client = "^0.22.1"
-btrfsutil = "^6.14"
+btrfsutil = "==6.13"
```

Comment:
Will remove it since maintaining this with these libs that are C based are complicated I think

I'm not using it in my newer filesystem support part